test/unit/fs-relative-emission/main/input.js

```diff
@@ -0,0 +1,11 @@
+
+const fs = require('fs');
+
+fs.readFile('../asset1.txt')
+
+fs.readFile('../blog/asset2.txt')
+
+const asset3 = 'asset3.txt';
+fs.readFileSync('../blog/author' + '/' + asset3, 'utf8');
+
+fs.readFile('../blog/author/../asset4.txt')
```

Comment:
```suggestion
fs.readFileSync('../asset1.txt')

fs.readFileSync('../blog/asset2.txt')

const asset3 = 'asset3.txt';
fs.readFileSync('../blog/author' + '/' + asset3, 'utf8');

fs.readFileSync('../blog/author/../asset4.txt')
```

Small suggestion since this code will fail when running `node input.js` right now.